tmt/utils/git.py

```diff
@@ -448,7 +448,7 @@ def validate_git_status(test: 'tmt.base.Test') -> tuple[bool, str]:
     """
     sources = [
         *test.fmf_sources,
-        test.fmf_root / '.fmf' / 'version'
+        test.anchor_path / '.fmf' / 'version'
```

Comment:
I think, here we should leave `fmf_root`. The `validate_git_status` function is used in `tmt teste export --how nitrate` for checking the test metadata are synced with the remote repo, so there definitely should be an `fmf` root where the test belongs.

Reply:
What if `fmf_root` is `None`? Which, as we know, would happen when running tmt outside of an fmf root. I suppose `test.fmf_root` should be tested, and if it's set, then added to the list. Its automatic presence is incorrect.

Reply:
Here, for this use case, it just cannot be `None`. Tests which are exported need to be stored on disk and have their `fmf` root defined (this is not like default plan or test metadata created under the `shell` discover step). So, if we want to double check it here, we can raise an exception at the beginning of the function.

Reply:
What about 6f41bc72? Does it look ok, @happz?

Reply:
Yep, or that. My point was, it can be `None` and that value must be handled somehow - at this moment, it was blindly added to the list, which we realized is not the right choice. A test & an exception are perfectly valid option.